src/List/ListItemAvatar.js

```diff
@@ -37,7 +48,11 @@ export default function ListItemAvatar(props, context) {
 }
 
 ListItemAvatar.propTypes = {
-  children: PropTypes.element.isRequired,
+  /**
+   * The content of the component, normally `Avatar`.
+   */
+  children: PropTypes.element.isRequired, // TODO: Should this component simply import Avatar and
```

Comment:
That's another approach. Why not, but it's deeply linked to the previous comment.